app/views/checkout/thankyou.scala.html

```diff
@@ -23,11 +23,15 @@
 <script type="text/javascript">
     guardian.pageInfo.slug="GuardianDigiPack:Order Complete";
     guardian.pageInfo.productData = {
+        // Omniture - TODO - fix the type field
```

Comment:
I need to talk to @JuliaBellis about what the enum values will be for non-digipack.